docs/common_links.txt

```diff
@@ -0,0 +1,60 @@
+.. These are ReST substitutions and links that can be used throughout the docs
```

Comment:
I think this should be a `.rst` file and not a `.txt` file.  It's already formatted as a RST file and my IDE is going mad with RST commands in a normal txt file.

Reply:
I just tried changing that and got an error that the substitution for `CustomParticle` is being defined twice.  I think what's happening is that this is being treated as a source file now when it shouldn't be.  A little less likely is that `rst_epilog`...which contains the contents of this file...is being appended to this file.  I'm attempting to add this file name to `exclude_patterns` in `conf.py`...